third_party/envoy-api.WORKSPACE

```diff
@@ -0,0 +1,16 @@
+workspace(name = "envoy_api")
```

Comment:
why exactly do we need this new workspace? AFAIK this is not something we're doing for other bazel dependencies.

Reply:
You are right. I was mistaken that Bazel needs a WORKSPACE file to treat a directory as external repository. But using `http_archive` automatically ensures that. So, this WORKSPACE has no use.